app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<h2>Log in</h2>
+
+<div class="flex-col form-logs">
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
+</div>
```

Comment:
- [ ] The log-in screen does not match the wireframe requested on the requirements.  Please, kindly make this view look more like the wireframe.

Your Implementation | Wireframe
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/173618538-1012cbcf-6d1e-45e4-b925-fa5183c61339.png) | ![image](https://user-images.githubusercontent.com/57421823/173618600-81acf353-4e89-4a7e-8555-c07b527b4925.png)

That way, we will meet this particular requirement 💪